sunpy/instr/rhessi.py

```diff
@@ -348,3 +391,183 @@ def _build_energy_bands(label, bands):
     unit = matched.group('UNIT').strip()
 
     return [f'{band} {unit}' for band in bands]
+
+
+def get_flare_list(start: str,
+                   end: str,
+                   source: str='NASA',
+                   file_format: str="hessi_flare_list_%Y%m.fits",
+                   inc=relativedelta(months=+1)):
+    """
+    Read and combine RHESSI flare lists from .fits files as specified with further parameters
+    Supported date formats are the same as in ``sunpy.time``.
```

Comment:
We have a way to add time types to the docs. 

See https://github.com/sunpy/sunpy/blob/1f79409372f8bd932c8f4db978e67bff451f04a7/sunpy/coordinates/sun.py#L45 for an example.

Reply:
I hope I used that somewhat correctly, but it seems it triggered an error in the tests (`timeseries_example.py`):

```
Warning, treated as error:
/home/circleci/project/examples/time_series/timeseries_example.py failed to execute correctly: Traceback (most recent call last):
  File "/home/circleci/project/.tox/build_docs/lib/python3.7/site-packages/sphinx_gallery/gen_rst.py", line 440, in _memory_usage
    out = func()
  File "/home/circleci/project/.tox/build_docs/lib/python3.7/site-packages/sphinx_gallery/gen_rst.py", line 425, in __call__
    exec(self.code, self.globals)
  File "/home/circleci/project/examples/time_series/timeseries_example.py", line 144, in <module>
    df_downsampled = ts_goes_trunc.data.resample('10T', 'mean')
  File "/home/circleci/project/.tox/build_docs/lib/python3.7/site-packages/pandas/core/generic.py", line 8102, in resample
    axis = self._get_axis_number(axis)
  File "/home/circleci/project/.tox/build_docs/lib/python3.7/site-packages/pandas/core/generic.py", line 407, in _get_axis_number
    raise ValueError(f"No axis named {axis} for object type {cls}")
ValueError: No axis named mean for object type <class 'pandas.core.frame.DataFrame'>
```

Reply:
That seems to be the new version of pandas that we had to update the example for. If you can merge in master, hopefully that should go away.

Reply:
Oh ok. Should I try to rebase my fork?

Reply:
You can either rebase the fork if you are comfortable doing that, or you can update the remote that points to this repository (if you have one) and then do `git pull <name> master`.

Reply:
phew. That resolved the error but introduced new ones :'(
also locally i cant do `import sunpy` anymore, it complains
```
File "C:\Projects\JHV\Python\sunpy\sunpy\__init__.py", line 18, in <module>
    from .version import __version__
ModuleNotFoundError: No module named 'sunpy.version'
```

it seems someone added `sunpy/version.py` to the gitignore..

Reply:
Yeah one of the changes in master, was to generate the version.py file at install. So you now how have to `pip install -e .` to create it. I am not sure we documented it anywhere (but it was how it used to be about 2 years ago).